src/core/wasm-module.js

```diff
@@ -124,7 +124,26 @@ class Impl {
  */
 
 /**
- * A pure static utility class which supports immediate and lazy loading of wasm modules.
+ * A pure static utility class which supports immediate and lazy loading of
+ * [WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly) modules. Note that you can
+ * load WebAssembly modules even before instantiating your {@link AppBase} instance.
+ *
+ * This class is generally only needed if you are developing against the Engine directly. Editor
+ * projects automatically load WebAssembly modules included in the project's assets.
+ *
+ * Do not use this class to load the Basis WebAssembly module. Instead, please refer to
+ * {@link basisInitialize}.
+ *
+ * @example
+ * // Load the Ammo.js physics engine
+ * pc.WasmModule.setConfig('Ammo', {
```

Comment:
If this is an engine-only example, do we still want to add pc proxy?

Reply:
All API reference example code prepends pc. before class names. I'm open to changing that, but only if it's globally consistent.